healpy/pixelfunc.py

```diff
@@ -906,13 +906,7 @@ def nside2npix(nside):
 
     >>> np.all([hp.nside2npix(nside) == 12 * nside**2 for nside in [2**n for n in range(12)]])
     True
-
-    >>> hp.nside2npix(7)
```

Comment:
Instead of removing this check, could we check that it returns the correct answer, 12*7**2 = 588?